nextflow_schema.json

```
@@ -87,150 +76,193 @@
         "run_ids": {
           "type": "string",
           "default": "All",
-          "description": "Specify a comma-separated list of run IDs to run. Use \"All\" to run all samples in the run_metafile."
+          "description": "Run IDs",
+          "help_text": "A comma-separated list of run IDs to run. Use \"All\" to run all samples in the run_metafile."
         },
         "project": {
           "type": "string",
-          "description": "Run samples only from a given project ID"
+          "description": "Project ID",
```

```suggestion
          "description": "Project ID(s)",
```
Maybe? Just to show that it can be a single or multiple IDs?